netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -6,19 +6,21 @@
 using Microsoft.Build.Utilities;
 using System;
 using System.Collections.Generic;
-using System.Diagnostics;
-using System.IO;
 
 namespace Microsoft.DotNet.Build.Tasks
 {
     public partial class HandlePackageFileConflicts : Task
     {
         private Dictionary<string, int> packageRanks = null;
+        private Dictionary<string, ConflictItem> externalRuntimeByFileName = new Dictionary<string, ConflictItem>(StringComparer.OrdinalIgnoreCase);
+        private Dictionary<string, ConflictItem> runtimeItemsByTargetPath = new Dictionary<string, ConflictItem>(StringComparer.OrdinalIgnoreCase);
 
         public ITaskItem[] References { get; set; }
         
         public ITaskItem[] ReferenceCopyLocalPaths { get; set; }
 
+        public ITaskItem[] ExternalRuntimeItems { get; set; }
```

Comment:
I'm not sure I understand what "external runtime" means.  Is there a better term we can use here?  For the shared framework, we usually use the term "platform", as in "type: platform". Or we also use the term "shared" since that is the name of the folder under 'dotnet'.